src/status_im/ui/screens/bottom_sheets/views.cljs

```diff
@@ -46,7 +49,24 @@
           (merge key-storage/migrate-account-password)
 
           (= view :pinned-messages-list)
-          (merge {:content pin.list/pinned-messages-list}))]
-    [bottom-sheet/bottom-sheet opts
-     (when content
-       [content (when options options)])]))
+          (merge {:content pin.list/pinned-messages-list})
+
+          (= view :drawer/reactions)
+          (merge {:content drawers/reactions}))]
+    (reagent/create-class
+     {:reagent-render         (fn []
+                                [bottom-sheet/bottom-sheet
+                                 (case view
+                                   :new-chat-bottom-sheet
+                                   (assoc opts :initial-height 150)
```

Comment:
could you elaborate? we shouldn't have any heights , it should be calculated in bottom sheet, it worked before

Reply:
it shouldn't have any default height, we need to implement it same way as old bottom sheet worked

Reply:
what do you mean ?

Reply:
yes height should be always be calculated from the content, if we want to adjust height it should be done in content height not in bottom sheet